processing/src/main/java/io/druid/query/aggregation/Histogram.java

```diff
@@ -58,6 +58,28 @@ public Histogram(float[] breaks, long[] bins, float min, float max)
     }
   }
 
+  public Histogram(Histogram other)
```

Comment:
The use of this constructor could be replaced by copyFrom() without adding ugly and questionable code like empty non-sense Histogram object. Replacing with `clone()` is possible, but I don't see why it's better. "Effective Java" advocates preferring copy constructors over `clone()`.